lib/metrics/eventLoopLag.js

```diff
@@ -16,7 +17,8 @@ module.exports = registry => {
 	const gauge = new Gauge({
 		name: NODEJS_EVENTLOOP_LAG,
 		help: 'Lag of event loop in seconds.',
-		registers: registry ? [registry] : undefined
+		registers: registry ? [registry] : undefined,
+		aggregator: AGGREGATORS.AVERAGE
```

Comment:
Should the `Gauge` class itself set this? So all get it for free? Or is there some use-case where it makes sense to not do so?